### ISSUE TITLE
[BUG] - Error using ParquetWriter in CUDF

### ISSUE THREAD

**Issue by hughcameron** [NVIDIA/spark-rapids#761]
**Describe the bug**
This pyspark workflow, provided in the [demo documentation](https://nvidia.github.io/spark-rapids/docs/examples.html), encounters an error when writing to Parquet.

**Steps/Code to reproduce bug**
I've implemented steps from [this documentation](https://nvidia.github.io/spark-rapids/docs/get-started/getting-started.html#spark-standalone-cluster) detailing how to launch a standalone Spark cluster. 

**Expected behavior**
The pyspark workflow to successfully output data to Parquet.

**Environment details (please complete the following information)**
- GPU: GeForce RTX 2080 Ti
- Running JupyterLab in container built from [this Dockerfile](https://github.com/hughcameron/spark_cuda/blob/master/Dockerfile)

**Additional context**
Stack trace from Spark UI:
```
java.lang.NoSuchMethodError: ai.rapids.cudf.ParquetWriterOptions$Builder.withMetadata(Ljava/util/Map;)Lai/rapids/cudf/WriterOptions$WriterBuilder;
	at com.nvidia.spark.rapids.GpuParquetWriter.(GpuParquetFileFormat.scala:202)
	at com.nvidia.spark.rapids.GpuParquetFileFormat$$anon$1.newInstance(GpuParquetFileFormat.scala:182)
	at org.apache.spark.sql.rapids.GpuSingleDirectoryDataWriter.newOutputWriter(GpuFileFormatDataWriter.scala:132)
	at org.apache.spark.sql.rapids.GpuSingleDirectoryDataWriter.(GpuFileFormatDataWriter.scala:117)
	at org.apache.spark.sql.rapids.GpuFileFormatWriter$.executeTask(GpuFileFormatWriter.scala:289)
	at org.apache.spark.sql.rapids.GpuFileFormatWriter$.$anonfun$write$16(GpuFileFormatWriter.scala:230)
	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:90)
	at org.apache.spark.scheduler.Task.run(Task.scala:127)
	at org.apache.spark.executor.Executor$TaskRunner.$anonfun$run$3(Executor.scala:446)
	at org.apache.spark.util.Utils$.tryWithSafeFinally(Utils.scala:1377)
	at org.apache.spark.executor.Executor$TaskRunner.run(Executor.scala:449)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by tgravescs** [comment]
it looks like either you aren't picking up the cudf jar or the version doesn't match what the spark rapids jar. If you are using spark rapids 0.2 version build from GitHub you can use the released cudf-0.15 jar, if you are using 0.1 make sure to use cudf-0.14 jar.

If you are running in standalone mode did you ensure your job is launched with: 
       --conf spark.executor.extraClassPath=${SPARK_CUDF_JAR}:${SPARK_RAPIDS_PLUGIN_JAR} \

In this case if you are launch Jupyter, make sure its notebooks get launched with that option as well.

If that doesn't solve your problem please provide more information - versions you are using, launch commands, etc.

**Comment by jlowe** [comment]
Agree with @tgravescs that this error is likely caused by a mismatched cudf version with the plugin.
@hughcameron please reopen if that ends up not being the case.
